### PR TITLE
(DOCSP-30565): Configure drone pipeline for `chat-server` staging

### PR DESCRIPTION
**Pull request by mongodben**
Configuration of Drone pipeline for chat-server staging builds. Will configure prod builds as we get closer to go live. 

This will work for development purposes. 

Automatically pushes the server to staging env when a change is added to `main` branch.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,43 +1,60 @@
...
+# ---
 kind: pipeline
 type: kubernetes
-name: test
+name: test-chat-server
 
 trigger:
   branch:
     - main
+    - DOCSP-30565 # TODO: remove later when ready to merge
```
Comment (mongodben) [inline .drone.yml:12]:
```suggestion
```

Delete before merge. Was for testing purposes.

**Review by cbush — APPROVED** [review]

File: chat-server/src/services/database.ts [hunk 2]
```diff
@@ -2,37 +2,37 @@ import { Db, MongoClient, ObjectId } from 'mongodb';
...
+export interface AssetDocument {
+  _id: string;
+  data: BinaryData;
+}
+
+interface PageDocument {
+  _id: ObjectId;
+  page_id: string;
+  filename: string;
+  ast: object;
```
Comment (cbush) [inline chat-server/src/services/database.ts:19]: Why is this needed?

We'll probably end up merging these definitions soon anyway, right?

Btw, TSC should complain about using `object` as such...

Comment (mongodben) [reply chat-server/src/services/database.ts:19]: this was just to get tsc to compile. 

will be deleted once we start populating with our own code.

and re usage of `object`, pls feel free to disallow that when you set up the tsconfig.
